app/assets/javascripts/assignments.js

```diff
@@ -25,7 +25,13 @@ $(document).ready(function() {
                             klass = 'btn-success';
                             break;
                     }
-                    $div.html('<a href="' + actions_map[mid].url + '" class="btn ' + klass + '" >' + actions_map[mid].action + '</a>')
+                    if ( actions_map[mid].action == 'Attempt' ){
```

Comment:
@jsyeo Do I need to fix this complains ? <code>actions_map</code> is an existing variable.